nebula/ui/components/VPNConnectionInfoContent.qml

```diff
@@ -78,19 +78,6 @@ VPNFlickable {
                     Layout.rightMargin: VPNTheme.theme.windowMargin
                 }
 
-                VPNCallout {
-                    calloutCopy: VPNl18n.ConnectionInfoMultiHopCallout
```

Comment:
are we still using this string. If not, should we remove it from strings.yaml?

Reply:
Opened PR #5440 to address this.